src/criteria/default/index.php

```diff
@@ -31,6 +31,53 @@
 		'category'          => 'reader_engagement',
 		'matching_function' => 'list__in',
 	],
+	'devices'                  => [
+		'name'        => __( 'Devices', 'newspack-popups' ),
+		'description' => __( 'The device the user is viewing the site on – e.g. mobile, tablet, etc.', 'newspack-popups' ),
+		'category'    => 'reader_engagement',
+		'options'     => [
+			[
+				'label'  => __( 'Desktop - 1280 px and above', 'newspack-popups' ),
+				'value'  => 'Desktop',
+				'params' => [
+					'max_width' => PHP_INT_MAX,
+					'min_width' => 1280,
+				],
+			],
+			[
+				'label'  => __( 'Laptop - 1024 px wide', 'newspack-popups' ),
+				'value'  => 'Laptop',
+				'params' => [
+					'max_width' => 1280,
+					'min_width' => 1024,
+				],
+			],
+			[
+				'label'  => __( 'Tablet - 768 px wide', 'newspack-popups' ),
+				'value'  => 'Tablet',
+				'params' => [
+					'max_width' => 1024,
+					'min_width' => 768,
+				],
+			],
+			[
+				'label'  => __( 'Mobile (large phones) - 480 px wide', 'newspack-popups' ),
+				'value'  => 'Mobile',
+				'params' => [
+					'max_width' => 768,
+					'min_width' => 480,
+				],
+			],
+			[
+				'label'  => __( 'Mobile (small phones) - 360 px wide', 'newspack-popups' ),
+				'value'  => 'Mobile small',
+				'params' => [
+					'max_width' => 360,
```

Comment:
There's a "hole" between "Mobile" and "Mobile small" -> if the screen size is between 360 and 480, no device will match it. But the list is out of sync with the breakpoints defined in the image in the PR description – 

| device | PR desc. | code |  
|--------------|-----------|-----------|
| Small mobile | <=360 | <=360 | 
| Large mobile | <=480 | <=768 | 
| Tablet | <=768 | <=1024 | 
| Laptop | <=1024 | <=1280 | 
| Desktop | >1280 | >1280 | 

So only Small mobile and Desktop match the requirements.

Reply:
You are right. That is off. But am I just confused or will those intervals never work? There will always be a gap on one end or the other? It's probably because I took it from the "Portrait" column and the original table has more context because it has landscape too?

Would this make more sense?
| Device Type   | Width (px) |
|---------------|------------|
| Desktop       | ≥ 1280     |
| Laptop        | 1024-1279  |
| Tablet        | 768-1023   |
| Phone         | 480-767    |
| Small Phone   | < 480      |

Reply:
Yeah, that makes sense. And you're right about the gap being there anyway – I didn't catch it at first.